qiskit/pulse/commands/_acquire.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+# pylint: disable=invalid-name,missing-docstring,missing-param-doc
```

Comment:
Can you check if the disables are really needed in all the files? At a quick glance, these lines seem to be mostly a copy-paste byproduct and they might not apply (and obscure potential warnings if present).

Reply:
Sorry, I just overlooked. I'll fix soon.